docs/faq.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: FAQ
+---
+
+* TOC
+{:toc}
+
+## What browser does best on web-platform-tests?
+
+This is a question we've strongly avoided answering: we don't want to
+publish any comparison tables as doing so has previously led to
+testsuites being used for marketing, which has then led to lower
+quality testsuites.
+
+For example, we've previously had:
+
+ * Vendors release only tests that they pass (with obvious omissions),
+ * Vendors release thousands of near-identical tests (increasing their
+   pass percentage and making it closer to other vendors who passed
+   the existing tests), and
+ * Vendors delay releasing tests until they have shipped a product
+   that passes them (leading to duplication of effort and other
+   vendors fail to correctly handle some edge-cases that the delayed
+   testsuite covered).
+
+## Why can't web-platform-tests be used for [caniuse.com][]?
+
+web-platform-tests provides a large number of tests for each standard,
```

Comment:
I think this is mostly OK, but we could mention that it's essentailly a special case of the issue above.